agentops/client.py

```diff
@@ -148,10 +104,9 @@ def record(self, event: Event):
             self._worker.add_event(
                 {'session_id': self._session.session_id, **event.__dict__})
         else:
-            logging.info("This event was not recorded because the previous session has been ended" +
-                         " Start a new session to record again.")
+            logging.warn("AgentOps: Cannot record event - no current session")
 
-    def record_action(self, event_name: str, tags: Optional[List[str]] = None):
+    def record_function(self, event_name: str, tags: Optional[List[str]] = None):
```

Comment:
Why `record_function`?

Reply:
We're using 'action' as an event type and record action was never really that great of a name. I'm open to other names